screens/auth/PasswordResetScreen.js

```diff
@@ -0,0 +1,266 @@
+import { FontAwesome5 } from '@expo/vector-icons';
+import { FirebaseRecaptchaVerifierModal } from 'expo-firebase-recaptcha';
+import * as firebase from 'firebase';
```

Comment:
Nit but I think this should be capitalized for consistency

Reply:
its lowercase to reflect an instance of an object